kong/db/schema/entities/routes.lua

```diff
@@ -198,7 +198,7 @@ local routes = {
         },
         values = {
           type = "array",
-          elements = typedefs.regex_or_plain_pattern,
+          elements = typedefs.header_regex_or_plain_pattern,
```

Comment:
The original design is that the value will be treated as a regex if and only if there is only one value and it starts with `~*`, does this implementation fit it?

Reply:
The number of elements is not limited. Let's reconsider how to do this